src/main/java/mil/dds/anet/database/OrganizationDao.java

```diff
@@ -95,6 +100,25 @@ public CompletableFuture<List<Organization>> getOrganizationsForPerson(
         personUuid);
   }
 
+  static class ResponsiblePositionsBatcher extends ForeignKeyBatcher<Position> {
+    private static final String sql =
+        "/* batch.getResponsiblePositionsForOrganization */ SELECT \"organizationResponsiblePositions\".\"organizationUuid\", "
+            + PositionDao.POSITIONS_FIELDS
+            + " FROM positions, \"organizationResponsiblePositions\" "
+            + "WHERE \"organizationResponsiblePositions\".\"organizationUuid\" IN ( <foreignKeys> ) "
+            + "AND \"organizationResponsiblePositions\".\"positionUuid\" = positions.uuid";
```

Comment:
Instead of querying two tables with an implicit JOIN here, consider adding an explicit JOIN instead.